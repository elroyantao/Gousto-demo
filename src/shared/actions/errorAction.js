
export const resetApiError = () => ({
  type: 'RESET_API_ERROR'
})
