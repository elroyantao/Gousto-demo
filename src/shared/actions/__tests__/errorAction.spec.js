import * as actions from '../errorAction'

describe('error actions', () => {
  it('resetApiError()', () => {
    expect(actions.resetApiError()).toMatchSnapshot()
  })
})
