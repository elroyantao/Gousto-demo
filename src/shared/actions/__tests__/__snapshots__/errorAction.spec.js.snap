// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`error actions resetApiError() 1`] = `
Object {
  "type": "RESET_API_ERROR",
}
`;
